api/test/shell/cli_test.sh

```diff
@@ -34,31 +47,49 @@ go build -o ./manager-api .
 sleep 3
 pkill -f manager-api
 
-if [[ ! -f "./logs/error.log" ]]; then
-    echo "failed: failed to write log"
-    exit 1
-fi
+check_logfile
 
-if [[ `grep -c "INFO" ./logs/error.log` -ne '0' ]]; then
+if [[ `grep -c "INFO" ${logfile}` -ne '0' ]]; then
     echo "failed: should not write info log when level is warn"
     exit 1
 fi
 
-#change level and path
+clean_logfile
+
+#change level
 
-sed -i 's/file_path: logs\/error.log/file_path: .\/error.log/' conf/conf.yaml
```

Comment:
please don't delete the existing case，it is for testing log filepath.thanks

Reply:
OK.